src/MarginTrading.MarketMaker/Services/Implementation/MarketMakerService.cs

```diff
@@ -65,34 +68,42 @@ public Task ProcessNewExternalOrderbookAsync(ExternalExchangeOrderbookMessage or
                 return Task.CompletedTask;
             }
 
-            var commands = new List<OrderCommand>
-            {
-                new OrderCommand {CommandType = OrderCommandTypeEnum.DeleteOrder}
-            };
-
-            foreach (var bid in resultingOrderbook.Bids)
-            {
-                commands.Add(new OrderCommand
-                {
-                    CommandType = OrderCommandTypeEnum.SetOrder,
-                    Direction = OrderDirectionEnum.Buy,
-                    Price = bid.Price,
-                    Volume = bid.Volume
-                });
-            }
+            var orderbooksToSend = _crossRatesService.CalcDependentOrderbooks(resultingOrderbook)
```

Comment:
Should be i think:
var orderbooksToSend = _crossRatesService.CalcDependentOrderbooks(resultingOrderbook)
                .Add(resultingOrderbook);
or transformations are not applied... (